src/Sarif.Viewer.VisualStudio/ProjectNameLookup.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Sarif.Viewer
+{
+    public sealed class ProjectNameLookup
+    {
+        private readonly static ProjectNameLookup _instance = new ProjectNameLookup();
+        private Dictionary<string, string> _projectNames = new Dictionary<string, string>();
+
+        static ProjectNameLookup() { }
+        private ProjectNameLookup() { }
+
+        public static ProjectNameLookup Instance => _instance;
+
+        public void SetName(string fileName)
+        {
+            if (_projectNames.ContainsKey(fileName))
+                return;
```

Comment:
>return [](start = 16, length = 6)

Our convention, as is the case with many MS teams, is always to use the braces:
```
if (condition)
{
    return;
}
```

Same with the if/else below.

Reply:
Easy fix!  I'll get that put in.